src/main/resources/templates/request_reservation.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html xmlns:th="www.thymeleaf.org">
+<head>
+  <meta charset="UTF-8">
+  <title>Reservation Details</title>
+</head>
+<body>
+    The details of your weekend get away at the <span th:text="${level}"></span> 
```

Comment:
Correctly displays the level, city, and email which were passed in as model attributes.